.github/actions/session_nonzero/action.yml

```diff
@@ -0,0 +1,49 @@
+name: Azure Nonzero session execution
+description: 'Execute a script as session >= 2.'
+
+inputs:
+  exec:
+    description: Script to execute
+    required: true
+  max-attempts:
+    description: Maximum number of attempts to acquire session > 2
+    default: "20"
+    required: false
+
+runs:
+  using: 'composite'
+  steps:
+    - name: Get session >= 2, and execute
+      shell: powershell
+      run: |
+        $higherSession=0
+        $count = 0
+        While ($higherSession -lt 2) {
+          Sleep 5
+          $count = $count + 1
+          If ($count -gt ${{ inputs.max-attempts }}) {
+            Write-Error "RDP never connected"
+            Exit (-1)
+          }
+
+          $sessions=$(C:\SysinternalsSuite\logonsessions.exe -nobanner | Select-String Session: | Sort-Object -Descending | Get-Unique)
+          If ( $sessions -ne $null -and $sessions.length -gt 0 ) {
+            $higherSession=$( $sessions )[0].ToString()[-1]
+          }
+        }
+
+        c:\sysinternalssuite\psexec.exe -nobanner -accepteula -w $PWD.Path -i $higherSession powershell.exe -Command @'
+        &{
+          ${{ inputs.exec }}
+        } 3>&1 2>&1 > .\test.log
```

Comment:
There is nothing equivalent of tee, we can’t get the output directly here? (I’m afraid of jobs that takes multiple minutes and no feedback on the action meanwhile…)

Reply:
Apparently there is a `Tee-Object`. I'll try it out.

Reply:
I get the same result as I used to, for some reason nothing is printed to stdout. I'll have to investigate a bit more.

Reply:
It fails because it is using RDP, so stdout/stderr are printed on the remote. I've quickly looked around the web and there seem to be ways of redirecting them from the remote to the local environment, but I'd leave that to the next PR

Reply:
interesting, I’m unsure to really understand why (for me, rdp was only for opening the session, but then, when you execute, your in another seat from the same session and get your output there), but that’s probably wrong :) anyway :+1: on addressing it in the next PR.